packages/credential-showcase-traction-adapter/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "credential-showcase-traction-adapter",
+  "version": "0.1.0",
+  "source": "src/index.ts",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "build": "tsc",
+    "build:clean": "tsc --build --clean && tsc --build",
+    "prettier": "prettier --write \"{packages,__tests__,!dist}/**/*.{ts,js,json,md}\"",
+    "test:ci": "jest --config=jest.json",
```

Comment:
can we not keep the tests in the main package.json?

Reply:
you can always just create a config if you want to test some individual tests

Reply:
fixed